src/metabase/lib/drill_thru/object_details.cljc

```diff
@@ -55,47 +50,3 @@
              (some? value))
     (object-detail-drill-for query stage-number context
                              (> (count (lib.metadata.calculation/primary-keys query)) 1))))
-
-(defmethod lib.drill-thru.common/drill-thru-info-method :drill-thru/pk
-  [_query _stage-number drill-thru]
-  (select-keys drill-thru [:many-pks? :object-id :type]))
-
-(defmethod lib.drill-thru.common/drill-thru-info-method :drill-thru/zoom
-  [_query _stage-number drill-thru]
-  (select-keys drill-thru [:many-pks? :object-id :type]))
-
-(defmethod lib.drill-thru.common/drill-thru-info-method :drill-thru/fk-details
-  [_query _stage-number drill-thru]
-  (select-keys drill-thru [:many-pks? :object-id :type]))
-
-(defmethod lib.drill-thru.common/drill-thru-method :drill-thru/pk
-  [query stage-number {:keys [column object-id]} & _]
-  ;; This type is only used when there are multiple PKs and one was selected - [= pk x] filter.
-  (lib.filter/filter query stage-number
-                     (lib.options/ensure-uuid [:field {} (lib.ref/ref column) object-id])))
-
-(defn- field-id [x]
-  (cond
-    (int? x)                   x
-    (string? x)                x
-    (and (vector? x)
-         (= :field (first x))) (field-id (nth x 2))
-    (map? x)                   (:id x)))
-
-(defmethod lib.drill-thru.common/drill-thru-method :drill-thru/fk
-  [query stage-number {:keys [column object-id]} & _]
-  (let [fk-column-id     (:fk-target-field-id column)
-        fk-column        (lib.metadata/field query fk-column-id)
-        fk-filter        (lib.options/ensure-uuid [:= {} (lib.ref/ref fk-column) object-id])
```

Comment:
Changed this to `(lib.filter/= fk-column object-id)`... no need to construct things the hard way